docs/content/dagster-cloud/deployment/serverless.mdx

```diff
@@ -79,7 +79,7 @@ You can also configure the `dagster-cloud` tool noninteractively; see [the CLI d
 Finally, deploy your project with Dagster Cloud Serverless:
 
 ```shell
-dagster-cloud serverless deploy \
+dagster-cloud serverless deploy-python-executable \
```

Comment:
bit unfortunate that the new default is the one with the non-default name (i.e. should `deploy` now be `deploy-image` or something?)

Reply:
I didn't want to break existing users. Going forward we can do this:
1. Add `deploy-image` that does docker deploy and tell users to migrate.
2. At some later point, change `deploy` to do `deploy-python-executable` instead of `deploy-image`
3. Later, remove `deploy-image`